interface/forms/newpatient/common.php

```diff
@@ -194,6 +197,8 @@ class="css_button link_submit" onClick="top.restoreSession()"><span><?php echo x
   "FROM openemr_postcalendar_categories where pc_active = 1 ORDER BY pc_seq ");
  while ($crow = sqlFetchArray($cres)) {
   $catid = $crow['pc_catid'];
+  // Show Thrapy group category only if global enable_group_therapy is true
+  if($catid == 1000 && !$GLOBALS['enable_group_therapy'])continue;
```

Comment:
for readability, place a space in front of continue (don't be ashamed to use spaces :) )